beesdoo_product/models/beesdoo_product.py

```diff
@@ -70,6 +70,11 @@ class BeesdooProduct(models.Model):
         compute="_compute_main_seller_id",
         store=True,
     )
+    main_seller_id_product_code = fields.Char(
+        string="Main Seller Product Code",
+        compute="_compute_main_seller_id",
+        store=True,
+    )
```

Comment:
That's a good way to do it. Maybe a better way would be to compute the top supplierinfo and have `main_seller_id` and `main_seller_id_product_code ` be related fields ?

```py
top_supplieinfo_id = fields.Many2one(
	comodel_name="product.supplierinfo", 
	compute=`_compute_main_seller_id`",
	store=True,
)
main_seller_id = fields.Many2one(
	comodel_name="res.partner", 
	related="top_supplieinfo_id.name
)
seller_product_code = fields.Char(
	related="top_supplierinfo_id.product_code"
)
```

What do you think about it ? Maybe add a todo ? Refactor it in Komunigi?

Reply:
Would be nice indeed. I will try to do it now. :)